### PR TITLE
libcerf: update source_urls

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic427 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1bad54bd87eeca1c3babe0ee020e34c6 for a full test report.

File: easybuild/easyconfigs/l/libcerf/libcerf-1.11-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -14,12 +14,9 @@ description = """
 toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
 toolchainopts = {'pic': True}
 
-source_urls = [
-    'http://apps.jcns.fz-juelich.de/src/libcerf/',
-    'http://apps.jcns.fz-juelich.de/src/libcerf/old',
-]
-sources = [SOURCE_TGZ]
-checksums = ['70101cac4a0d7863322d4d06cf95c507a9cfd64fc99ad1b31a8425204cfd9672']
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v1.11/']
```
Comment (boegel) [inline easybuild/easyconfigs/l/libcerf/libcerf-1.11-GCCcore-7.3.0.eb]: @smoors Please use `%(version)s` so this won't need to be touched if `--try-software-version` is used for future releases.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3129.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c0835be20df163d3565ccc3c279152d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4eb93b2a9b9966cf932c865b8f457cba for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**